### ISSUE TITLE
关于D_t的计算与论文中有所不同

### ISSUE THREAD

**Issue by ge-bin-hui** [EnVision-Research/ScalingAR#1]
您好！
在您论文中的4.1.2节CONDITIONAL CHANNEL: TEXT UTILIZATION STRENGTH中关于D_t的计算是使用rolling z-score normalization并通过clip来将其数值范围控制在[0,1]。

<img width="1006" height="481" alt="Image" src="https://github.com/user-attachments/assets/142be918-96c5-49e7-a837-bf6f9ea8b6aa" />

这与代码中的操作有所不同

`

            D_t = 0.0
            if uncond_logit is not None:
                K_t = float(kl_between_logits(cond_logit, uncond_logit).detach().cpu().item())
                running_K_min = min(running_K_min, K_t)
                running_K_max = max(running_K_max, K_t)

                if running_K_max > running_K_min:
                    D_t = max(0.0, min(1.0, (K_t - running_K_min) / (running_K_max - running_K_min + 1e-12)))
                else:
                    D_t = 0.0

                traj.D_bar = (1 - lambda_d) * traj.D_bar + lambda_d * D_t
                traj.D_history.append(D_t)
            else:
                traj.D_history.append(0.0)
`
请问这两种方式是等价的吗

**Comment by HaroldChen19** [comment]
Hi Binhui,

Thank you for reaching out.

In our practice, z-score and min-max yield similar results, and either strategy is acceptable. Apologies for just noticing the issue; we'll update the z-score deployment code asap.

Thanks again for reaching out and feel free to re-open the issue if there are further questions.

Best,
Harold
